agency/README.md

```diff
@@ -378,7 +378,7 @@ The reservation method enum describes the different ways in which a passenger ca
 | distance              | Float         | Conditionally Required | Total distance of the trip in meters (required if trip was completed)                                                                                                                                                                                                                                              |
 | cancellation_reason   | string        | Conditionally Required | The reason why a *driver* cancelled a reservation. (required if a driver cancelled a trip, and a `driver_cancellation` event_type was part of the trip)                                                                                                                                                            |
 | fare                  | [Fare](#fare) | Conditionally Required | Fare for the trip (required if trip was completed)                                                                                                                                                                                                                                                                 |
-| accessibility_options | Enum[]        | Optional               | The **union** of any accessibility options requested, and used. E.g. if the passenger requests a vehicle with `wheelchair_boarding`, but doesn’t utilize the features during the trip, the trip payload will include `accessibility_options: ['wheelchair_boarding']`. See [accessibility-options][accessibility-options] |
+| accessibility_options | Enum[]        | Optional               | The **union** of any accessibility options requested, and used. E.g. if the passenger requests a vehicle with `wheelchair_accessible`, but doesn’t utilize the features during the trip, the trip payload will include `accessibility_options: ['wheelchair_accessible']`. See [accessibility-options][accessibility-options] |
```

Comment:
What is supposed to be in the payload if the accessibility option is requested AND is used?

The link "accessibility-options" redirects to the agency spec, but that doesn't contain any mention of accessibility-options

Reply:
The link ought to redirect to `general-information.md`, hm. In the case that it's requested *and* used, it's inclusive, so it should just be the same as if it was only requested.